src/app/common/NotificationIcon.vue

```diff
@@ -5,8 +5,9 @@
     @click="openModal"
   >
     <KIcon
-      color="var(--yellow-300)"
       icon="notificationBell"
+      color="var(--yellow-300)"
+      hide-title
```

Comment:
Kind of a consequence of kongponents I think, but did we lose the `Notifications` word out of the HTML here? Quick aria-label or visually hidden span to put it back?

Reply:
I’ve opted to removes `hide-label` here because this doesn’t have a good label otherwise, yeah.

Reply:
Sounds good!